diff.py

```diff
@@ -1575,6 +1576,12 @@ def __init__(self, config: Config) -> None:
         super().__init__(config)
         self.seen_jr_ra = False
 
+    def _normalize_arch_specific(self, mnemonic: str, row: str) -> str:
+        if self.config.ignore_equivalent_immediates and mnemonic == "li":
+            regex = re.compile(f",0x([1-9])$")  # only 1 thru 9
```

Comment:
The idea of compiling a regex is to compile it only once, not every time the function is called

Reply:
I was googling and read:

> **Is it worth using Python’s re.compile()?**
 As you know, Python always internally compiles and caches regexes whenever you use them anyway (including calls to search() or match()), so using compile() method, you’re only changing when the regex gets compiled.

But sure, I'll stick it outside the function